include/swift/ABI/MetadataValues.h

```diff
@@ -1925,8 +1925,10 @@ class JobFlags : public FlagSet<size_t> {
 
     // Kind-specific flags.
 
-    Task_IsChildTask  = 24,
-    Task_IsFuture     = 25
+    Task_IsChildTask    = 24,
+    Task_IsFuture       = 25,
+    // Task_IsGroup       = 26,
+    Task_HasLocalValues = 27
```

Comment:
Representing it as fragment we can save some space -- many tasks will not (nor their parents) declare task locals and we can skip the fragment entirely then.